dist/ci/travis_script.sh

```diff
@@ -46,15 +39,16 @@ if test -z "$SUBTEST"; then
       pushd ../backend
       make test_unit
       ;;
+    migrate)
+      mv db/structure.sql db/structure.sql.git
+      bundle exec rails db:drop db:create || exit 1
+      xzcat test/dump_2.5.sql.xz | sed '/^USE api_.*;/d' | mysql ci_api_test
```

Comment:
> The real scenario is that people used the old distro, with old mysql and old ruby stack to create their OBS 2.5 installation. You get different result when using the migration 001 in new environemt on new OS (using new mariadb, new rails stack and so on)
  
I think this is an entirely different thing you are describing. You are saying that this test should effectively check that you can upgrade from 2.5 to current stable version. If you want to do that somewhere else thats fine but I think thats beyond the scope of this travis test. Here we should just check that the migrations run from the 001 migration and that it matches structure.sql. Not worry about earlier version of mysql/ruby.